tests/testthat/test_corr_bestlag.R

```diff
@@ -12,8 +12,10 @@ test_that("the corr is", {
                timepoints = c(0, 2, 4, 6, 8, 18, 24, 32, 48, 72), penalty = "high")$lags[1]), 1)
   expect_equal(signif(corr.bestlag(data = simdata[20:30, ],
                timepoints = c(0, 2, 4, 6, 8, 18, 24, 32, 48, 72))$corr[5], 2), 0.91)
-  expect_error(signif(corr.bestlag(data = simdata,
-               timepoints = c(0, 2, 4, 6, 8, 18, 24, 32, 48, 72, 75))$corr[10], 2))
+  expect_equal(signif(corr.bestlag(data = simdata[20:30, ],
```

Comment:
The new lines 15-16 look the same as lines 13-14.  Did you mean to delete 13-14 and leave the test case that was deleted here?